recipe/meta.yaml

```
@@ -34,6 +35,7 @@ outputs:
         - ninja
         - make  # [unix]
         - cmake-no-system
+        - zlib
       host:
         - zlib
```

```suggestion
        - zlib
        - liblzma-devel
```
Could NOT find LibLZMA (missing: LIBLZMA_LIBRARY LIBLZMA_INCLUDE_DIR